TheraLang.Web/Controllers/PaymentHistoryController.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AutoMapper;
+using Common;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using TheraLang.BLL.DataTransferObjects;
+using TheraLang.BLL.Interfaces;
+using TheraLang.Web.ViewModels;
+
+namespace TheraLang.Web.Controllers
+{
+    [Route("api/paymentHistory")]
```

Comment:
if all your methods have to be [Authorize], you can use this attribute one more time here
it is also ok, but more code)

Reply:
thnx